### ISSUE TITLE
Enhance configuration file

### ISSUE THREAD

**Issue by wusitee** [hey2022/tls-xb#51]
**Is your feature request related to a problem? Please describe.**
Since there are more and more data and statistics which tls-xb is able to display, using a configuration 
file to manage and simplify it would be useful.

**Describe the solution you'd like**
Add more options in config.toml

**Describe alternatives you've considered**
make more useful arguments like `--table` for using table to manage outputs

**Comment by lanbinleo** [comment]
Yes, we should provide more options for user, to let them customize their result in the future.

**Comment by hey2022** [comment]
Would this be resolved by #97 or is there a specific feature wanted in the config file?

**Comment by wusitee** [comment]
> Would this be resolved by [#97](https://github.com/hey2022/tls-xb/pull/97) or is there a specific feature wanted in the config file?

I think it is solved as we now have a proper configuration file, though there are just few options.
